babel.config.js

```diff
@@ -20,5 +20,10 @@ module.exports = {
     '@babel/plugin-proposal-class-properties',
     '@babel/proposal-object-rest-spread',
     'babel-plugin-styled-components'
-  ]
+  ],
```

Comment:
With us using `data-test-*` IDs more in our components we should probably remove these attributes in our production bundles.